lib/demos/material/menu_demo.dart

```diff
@@ -192,10 +188,7 @@ class _SectionedMenuDemo extends StatelessWidget {
 // a menu whose current value is highlighted and aligned over the
 // list item's center line.
 class _SimpleMenuDemo extends StatefulWidget {
-  const _SimpleMenuDemo({
-    Key? key,
```

Comment:
this has been removed from private classes which were not providing keys, due to unused element lint. Although they are actually not being used, I am not sure if this is the style.